src/lib/ketrew_configuration.mli

```diff
@@ -180,3 +142,47 @@ val connection: client -> string
 val token: client -> string
 
 val standalone_of_server: server -> standalone
+
+
+val load_exn:
+  ?and_apply:bool ->
+  ?profile:string ->
+  [ `From_path of string
+  | `Guess
+  | `In_directory of string
+  | `Override of t ] ->
+  t
+(** Load a configuration.
```

Comment:
This documentation is confusing.
- I think describing the optional arguments later would be easier,.
- Why don't the order of the arguments match?
- `` `Override c `` seems super awkward as an argument to `load_exn`